CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Version 0.5.1-dev (changes since 0.5.1 go here)
 
 ### Features
+* Added ``skbio.tree_tree._cache_leafcount`` ``skbio.tree_tree._cache_height`` and ``skbio.tree_tree._cache_depth`` for calculating leaf count in subtrees, subtree height, and node depth. This is particularly useful for calculating tree coordinates for visualization ([#1514](https://github.com/biocore/scikit-bio/pull/1514))
```

Comment:
Adding an inline comment on @jairideout's request of asserting the PyCogent contributors are noted in skbio's pre history, and that PyCogent attribution is also noted